### ISSUE TITLE
Add support for PHP 7

### ISSUE THREAD

**Issue by lmakarov** [blinkreaction/drude#115]
Looks like PHP 7 really rocks compared to PHP 5.6. and even outperforms HHVM
http://www.jeffgeerling.com/blogs/jeff-geerling/benchmarking-drupal-8-php-7-vs-hhvm

It's time to get back to PHP multi-version support and add either a separate [cli image](https://github.com/blinkreaction/docker-drupal-cli) version with PHP 7 or put the two side-by-side in a single image.

**Comment by lmakarov** [comment]
Some high level testing using `time drush si -y`

**Drupal 7**

```
php5.6 w/ xdebug: 0m6.694s
php7.0 w/ xdebug: 0m5.282s
php5.6: 0m5.590s
php7.0: 0m4.385s
```

**Comment by lmakarov** [comment]
Released in [v1.4.0](https://github.com/blinkreaction/drude/releases/tag/v1.4.0)
Documentation: https://github.com/blinkreaction/drude/blob/v1.4.0/docs/settings.md#using-different-php-versions

**Comment by kplanz** [comment]
Nice! Just one note: I had to change this line in the dsh exec from `php5-fpm` to `php-fpm` to get it working: https://github.com/blinkreaction/drude/blob/develop/bin/dsh#L1526

@lmakarov not sure how this could be handled correctly in the future. Probably rename `php5-fpm` to `php-fpm` also in here to have it consistent: https://github.com/blinkreaction/docker-drupal-cli/blob/develop/config/supervisord.conf#L5

**Comment by lmakarov** [comment]
@kplanz thanks for the reminder. I came to a conclusion that renaming `php5-fpm` to `php-fpm` is the only reasonable solution. Unfortunately this is also a breaking change, so folks will have to update to the latest dsh version when we push this out, as well as pull the most recent `cli` image.

We could also add some checks in dsh to support legacy configuration.

Going to reopen this.

**Comment by lmakarov** [comment]
Pushed a fix to develop ^^

**Comment by lmakarov** [comment]
Released in [v1.4.1](https://github.com/blinkreaction/drude/releases/tag/v1.4.1)
